DEVELOPMENT.md

```diff
@@ -10,6 +10,11 @@ We recommend you use IntelliJ as your IDE. The code style template for the proje
 * Ensure that all files have the appropriate license header; you can generate the license by running `mvn license:format`.
 * Consider using String formatting (printf style formatting using the Java `Formatter` class): `format("Session property %s is invalid: %s", name, value)` (note that `format()` should always be statically imported). Sometimes, if you only need to append something, consider using the `+` operator.
 * Avoid using the ternary operator except for trivial expressions.
+* It is allowed to declare public members in private inner classes if they are part of their API.
```

Comment:
`It is allowed to declare public members`

`public and protected`?

Reply:
Actually I think it is suggested. Not only allowed. To distinguish public interface of private class from int internal implementation.